DNN Platform/Website/Default.aspx.cs

```diff
@@ -645,6 +653,110 @@ private void InitializePage()
             }
         }
 
+        private string GenerateCssCustomProperties()
+        {
+            string cacheKey = $"Dnn_Css_Custom_Properties_{this.PortalSettings.PortalId}";
+            string cache = Common.Utilities.DataCache.GetCache<string>(cacheKey);
+
+            if (!string.IsNullOrEmpty(cache))
+            {
+                return cache;
+            }
+
+            var styles = this.PortalSettings.Styles;
+            var sb = new StringBuilder();
+            sb
+                .AppendLine(@"<style type=""text/css"">")
+                .AppendLine(@":root {")
+                .AppendLine($"--dnn-color-primary: #{styles.ColorPrimary.MinifiedHex};")

```

Comment:
If we're using a `StringBuilder`, we probably want to use it to add the values, rather than string interpolation

Reply:
You mean like this?

```
  .AppendFormat("--dnn-color-secondary-dark: #{0};\n", styles.ColorSecondaryDark.MinifiedHex)
```

I kinda find it less readable. Is this a performance concern or did I misunderstand the idea ?

Reply:
Yes, I'm thinking about performance. You should be able to do it all as a single multi-line interpolated string, and get rid of the `StringBuilder`.

Reply:
Yeah, good idea... I`ll revisit this

Reply:
Done in https://github.com/dnnsoftware/Dnn.Platform/pull/5851/commits/1b8b566f9f2c7aeb4d5d43fa657a8ed0919a1df6